Documents/idea document 2/ideadocument2.md

```diff
@@ -7,8 +7,38 @@
 # Methods
 (Thom)
 
-# Speficications & Execution Plan
+# Specifications & Execution Plan
 
 # Expected Outcomes
 
 # Evaluation & Outlook
+
+## Evaluation
+In the section Specifications & Execution Plan a list of MoSCoW Specifications was presented.
+These metrics are a useful and structured manner to evaluate wether or not the application is working as intended.
+The MoSCoW specifications also describe the importance of each feature.
+Each feature can then be checked if they are implemented and how well they are working.
+The must haves are the most important features and are the bare minimum for the application to present the findings on the neighborhoods.
+The should haves are expanding features on the basic functionality.
+They can be inspected on how well they filter and modify the presented data to the user.
+The could haves describe extra functionalities that can be implemented after the rest of the application is implemented properly.
+
+Besides functional evaluation of the project by the MoSCoW specifications, the project can also be evaluated by testing how well the application performs in different user scenarios.
+In the following table user scenarios for which Amsterdam In Motion could be used in will be described next to the possible relevant stakeholders.
+
+| Stakeholder         | User Scenario                                                                                                                                                                                                |
+|---------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Local Government    | The government wants to know in what area of the city there is a need for more streetlights for runners. The government wants to know what area would be most suitable and in need of a new sports complex.  |
+| Entrepeneurs        | A company wants to know where to build their new football/hockey store.                                                                                                                                      |
+| Individual Athletes | An athlete wants to know on what time of a day of the week the park is least busy. This way the athlete can plan their exercise accordingly.                                                                 |
+
```

Comment:
I think if you introduce a table for evaluating, there needs to be room for putting results in the table. Not sure though